src/app/modules/sgid/sgid.controller.ts

```diff
@@ -61,7 +87,12 @@ export const handleLogin: ControllerHandler<
 
   const jwtResult = await SgidService.retrieveAccessToken(code)
     .andThen((data) => SgidService.retrieveUserInfo(data))
-    .andThen(({ data }) => SgidService.createJwt(data, rememberMe))
+    .andThen(({ data }) =>
+      SgidService.createSgidSingpassJwt(
+        { 'myinfo.nric_number': data['myinfo.nric_number'] },
```

Comment:
Would be good opportunity to switch to using `SGID_MYINFO_NRIC_NUMBER_SCOPE` here instead of magic string :)